acceptance/tests/options/config_file/cache_migrate_from_facter_3.rb

```diff
@@ -0,0 +1,87 @@
+test_name 'ttls configured custom facts files creates cache file and reads cache file' do
```

Comment:
the test name doesn't correspond with what the test actually does

Reply:
Updated